src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Administer.cshtml.cs

```diff
@@ -98,9 +103,9 @@ public IActionResult OnPostDeleteSuggestions()
 
     public IActionResult OnPostImportRedirects()
     {
-        if (ImportFile == null || !ImportFile.IsXml())
+        if (ImportFile == null)
         {
-            Message = "The uploaded file is not a valid XML file. Please upload a valid XML file.";
+            Message = "The uploaded file is not a valid XML or CSV file. Please upload a valid XML/CSV file.";
```

Comment:
This can be moved into a separate method.

`private IAction BuildResponse(string message, CardType cardType)`